micro-ui/web/packages/modules/workbench/src/configs/UICustomizations.js

```
@@ -1009,5 +1010,116 @@ export const UICustomizations = {
+        });
+        return res;
+      };
+      const fun = async () => {
+        res = await callSearch();
+      };
+      // fun();
+      const [showToast, setShowToast] = useState(null);
+      switch (key) {
+        case "HIERARCHY_NAME":
+          return row?.hierarchyType;
+          break;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)